src/scripts/services/dimStoreService.factory.js

```diff
@@ -743,6 +743,7 @@ function StoreService(
       visible: true,
       sourceHashes: itemDef.sourceHashes,
       lockable: normalBucket.type !== 'Class' && ((currentBucket.inPostmaster && item.isEquipment) || currentBucket.inWeapons || item.lockable),
+      taggable: item.lockable && !item.isEngram,
```

Comment:
It's a function of the item you're building `createdItem` via `ItemProto` - not the raw Bungie item data. And even if it were, you'd have to call that function, no?